src/test/java/seedu/address/logic/commands/UpdateSessionRemarkCommandTest.java

```diff
@@ -53,6 +55,18 @@ public void execute_validSessionNumberAndSessionRemark_success() throws CommandE
         assertEquals(expectedSession, updatedSession);
     }
 
+    @Test
+    public void execute_invalidSessionNumber_throwsCommandException() {
```

Comment:
throws `SessionNotFoundException` ? not `CommandException`

Reply:
youre right